### PR TITLE
MVP-329: Standard json response

### PR DESCRIPTION
**Pull request by BradyMitch**
## 🎯 Summary

  
[MVP-329](https://citz-imb.atlassian.net/jira/browse/MVP-329)

- Adds a standard json response for express requests. We can expand on it further, but this is a start.

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](/CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -49,10 +49,29 @@ export type ZodValidationErrorDetail = {
   message: string;
 };
 
+export type StandardResponseInput = {
+  success: boolean;
+  data: object;
+  message?: string;
+};
```
Comment (SDToews) [inline src/types.ts:56]: consider using generic type here

```suggestion
export type StandardResponseInput<TData> = {
  success: boolean;
  data: <TData>;
  message?: string;
};
```

File: src/types.ts [hunk 2]
```diff
@@ -49,10 +49,29 @@ export type ZodValidationErrorDetail = {
...
+
+export type StandardResponse = {
```
Comment (SDToews) [inline src/types.ts:58]: Generic would go here too

**Review by SDToews — CHANGES_REQUESTED** [review]
I ran the tests and 8 suites failed.  Looks like it's mostly because of implicit 'any' types

File: src/types.ts (hunk 1 above)
Comment (BradyMitch) [reply src/types.ts:56]: Im not sure generics is what we want here, and theres really no reason for us to be type checking it other than it being an object.

**Review by SDToews — APPROVED** [review]

**Pull request merged**